packages/land/contracts/Land.sol

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: MIT
+/* solhint-disable no-empty-blocks */
+pragma solidity 0.8.23;
+
+import {LandBaseToken} from "./mainnet/LandBaseToken.sol";
+import {ERC721BaseToken} from "./mainnet/ERC721BaseToken.sol";
+import {OperatorFiltererUpgradeable} from "./mainnet/OperatorFiltererUpgradeable.sol";
+import {IOperatorFilterRegistry} from "./mainnet/IOperatorFilterRegistry.sol";
+import {IERC2981Upgradeable} from "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+import {IRoyaltyManager} from "@sandbox-smart-contracts/dependency-royalty-management/contracts/interfaces/IRoyaltyManager.sol";
+
+/**
+ * @title Land Contract
+ * @author The Sandbox
+ * @notice LAND contract
+ * @dev LAND contract implements ERC721, quad and marketplace filtering functionalities
+ */
+contract Land is LandBaseToken, OperatorFiltererUpgradeable {
+    uint16 internal constant TOTAL_BASIS_POINTS = 10000;
+
+    IRoyaltyManager private royaltyManager;
+    uint8 private initialized = 3;
```

Comment:
no way to remove the hardcoded 3 ?

Reply:
It can be removed if we're mentioning version for the first time.